package de.cacheoverflow.veloxio.network.exceptions

class EpollException(message: String, cause: Throwable) : RuntimeException(message, cause)
